mw-webapp/src/logic/aboutProjectPage/AboutProjectPage.module.scss

```diff
@@ -8,3 +8,19 @@
 .titleAccordion {
   text-align: center;
 }
+
+.titlePage {
+  position: absolute;
+  top: 1rem;
+  left: 44%;
+  color: $lightyellow;
+  text-shadow: 2px 2px 2px $button-shadow;
+}
+
+.titleWhenLoggedIn {
```

Comment:
Are you sure that we need so many title styles? We don't have design yet

Reply:
Are you sure that  we need different styles for one title (titlePage, titleWhenLoggedIn, etc) It's too amny styles for just one title.